dist/database.js

```diff
@@ -0,0 +1,259 @@
+'use strict';
+var __extends = (this && this.__extends) || (function () {
```

Comment:
If we are going to TypeScript, we will not push the `dist` folder to the repo. Please add it to the `.gitignore`.